docs/reference/buildx_create.md

```diff
@@ -141,7 +141,7 @@ to achieve that.
 
 Passes additional driver-specific options.
 
-Note: When using quoted values for example for the `nodeselector` or
+Note: When using quoted values for example for the `nodeselector`, `annotations` or
```

Comment:
```suggestion
Note: When using quoted values for example for the `nodeselector`, `annotations`, `labels` or
```

Reply:
I was expecting this comment :)

Should I remove `for example` from the sentence, since it's supposed to be all-inclusive?

```suggestion
Note: When using quoted values for the `nodeselector`, `annotations`, `labels` or
```

Reply:
(note: I applied my suggestion)